.github/workflows/test-stellar.yaml

```diff
@@ -0,0 +1,236 @@
+              --network-passphrase "$NETWORK_PASSPHRASE"
+
+      - name: Prepare local.json
+        run: |
+          echo '{
+              "chains": {
+                  "stellar": {
+                      "name": "Stellar",
+                      "axelarId": "stellar",
+                      "networkType": "local",
+                      "chainType": "stellar",
+                      "tokenSymbol": "XLM",
+                      "rpc": "http://127.0.0.1:8000/soroban/rpc",
+                      "horizonRpc": "http://127.0.0.1:8000",
+                      "contracts": {}
+                  }
+              }
+          }' > ./axelar-chains-config/info/local.json
+
+      - name: Display local.json
+        run: cat ./axelar-chains-config/info/local.json
+
+      ###### Acquire WASM Contracts   ######
+      - name: Configure CF credentials
+        run: |
+          cd $HOME; mkdir ~/.aws; touch ~/.aws/credentials; touch ~/.aws/config
+          echo "[default]
+          aws_access_key_id = ${{ secrets.cf-bucket-access-key-id }}
+          aws_secret_access_key = ${{ secrets.cf-bucket-secret-access-key }}" > ~/.aws/credentials
+          echo "[default]
+          region=auto
+          output=json" > ~/.aws/config
+
+      - name: Fetch WASM contracts from CF
+        env:
+          S3_BUCKET_NAME: ${{ vars.CF_BUCKET_NAME }}
+          ENDPOINT_URL: ${{ secrets.cf-endpoint-url }}
+          CF_BUCKET_ROOT_KEY: ${{ vars.CF_BUCKET_ROOT_KEY }}
+        run: |
+          export CF_WASM_BUCKET_ROOT_KEY="${CF_BUCKET_ROOT_KEY}/${{ env.PACKAGE_NAME }}/${{ env.PACKAGE_VERSION }}/wasm"
+          for name in axelar_gateway axelar_operators axelar_gas_service; do
+            aws s3api get-object --bucket $S3_BUCKET_NAME --key "$CF_WASM_BUCKET_ROOT_KEY/$FILE_KEY" --body "$name.optimized.wasm" --acl public-read --endpoint-url $ENDPOINT_URL
+          done
+
+          ls
+
+          WASM_DIR=./
+          echo "WASM_DIR=${WASM_DIR}" >> $GITHUB_ENV
+
+      - name: Setup Stellar wallet
+        run: |
+          stellar keys generate test \
+              --rpc-url "$RPC_URL" \
+              --network-passphrase "$NETWORK_PASSPHRASE"
+
+          STELLAR_PRIVATE_KEY=$(stellar keys show test)
+          STELLAR_ADDRESS=$(stellar keys address test)
+          echo "STELLAR_PRIVATE_KEY=${STELLAR_PRIVATE_KEY}" >> $GITHUB_ENV
+          echo "STELLAR_ADDRESS=${STELLAR_ADDRESS}" >> $GITHUB_ENV
+
+      - name: Fund Stellar wallet
+        run: node stellar/faucet --recipient $STELLAR_ADDRESS
+
+      - name: Prepare .env
+        run: |
+          echo "PRIVATE_KEY = '$STELLAR_PRIVATE_KEY'" >> .env
+          echo "ENV = 'local'" >> .env
+          echo "YES = 'true'" >> .env
+
+      ###### Command: Deploy Contract ######
+      - name: Deploy axelar_gateway
+        run: |
+          CONTRACT=axelar_gateway
+          node stellar/deploy-contract deploy $CONTRACT --wasm-path $WASM_DIR/$CONTRACT.optimized.wasm --initialize
```

Comment:
`--initialize` parameter has been removed since we started using `__constructor`. please refer to the README: https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/stellar/README.md